apollo-integration/src/test/java/com/apollographql/apollo/IntegrationTest.java

```diff
@@ -211,6 +209,14 @@
     assertThat(body.hasErrors()).isFalse();
   }
 
+  @Test(expected = ApolloException.class) public void fieldMissing() throws Exception {
```

Comment:
@sav007 I changed this to be an exception. This should never happen per the GQL spec, so seems "exceptional".